### ISSUE TITLE
Fix (Feat?): Bump is not made depending on scope convention

### ISSUE THREAD

**Issue by armand1m** [leonardoanalista/corp-semantic-release#50]
Hi. First of all, let me thank you for creating this module. It fits my needs in private repositories really well.
However, I use to write my commit messages like this:

`fix(component/sidebar): Fixes onChange event`
`feat(component/header): Adds <HeaderComponent>`

When using this kind of convention in the scope, the module just ignores it and doesn't bump a new release. When not using `/`, it works really well.

**Comment by leonardoanalista** [comment]
Hi Armando, this sounds like a bug. It should work if your scopes contain slashes.
We'll have to look at this. Feel free to send a PR if you like. Otherwise I (or anyone else) will get on this when possible.

**Comment by uglow** [comment]
@armand1m Looking at a few other libraries that parse commits (such as https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-commits-parser/index.js#L9), you can see that the commit message RegEx for the scope `[\w\$\.\-\* ]` only allow ASCII letters & numbers & underscore (`[A-Za-z0-9_]`), `$`, `.`, `-`, `*`, and ` ` (space) characters. `/` is not allowed. 
The risk of changing the behaviour for this package is that one of our downstream dependencies (e.g. `conventional-changelog`) may not work with this character. And if you change to another tool and it parses your commits with `/` in them, it may not work either.

In short, the `/` should not be used in the scope of the commit message.
